assets/css/utilities/_typography.scss

```diff
@@ -0,0 +1,89 @@
+.type-xs { font-size: .75em; }
```

Comment:
Please remove this file. If you have a change _typography.scss please make a PR to https://github.com/blockchain/brand-assets/blob/css/sass/utilities/_typography.scss